### DISCUSSION TITLE
New Auth0 SDK in quasar Failing.

### DISCUSSION THREAD

**Discussion by brogits** [quasarframework/quasar#16894] [General - Components / Directives / etc] [answered]
I am just wondering if the new Auth0 SDK for vue is now supported in quasar.

`import { createAuth0 } from '@auth0/auth0-vue';`

My boot file looks like this:
```
import { boot } from 'quasar/wrappers';
import { createAuth0 } from '@auth0/auth0-vue';

export default boot(async ({ app }) => {
  app.use(
    createAuth0({
      domain: AUTH0_DOMAIN,
      clientId: AUTH0_CLIENT_ID,
      authorizationParams: {
        redirect_uri: AUTH0_CALLBACK_URL,
      },
    })
  );
....
```

and in quasar config I have ` boot: ['auth0', 'axios'],`

Error:

`[Quasar] boot error: SyntaxError: The requested module '/node_modules/.q-cache/vite/pwa/deps/@auth0_auth0-vue.js?v=e05013a5' does not provide an export named 'default' (at axios.js?t=1707674681237:3:8)`

**Comment by yusufkandemir** [comment, marked as answer]
Please share `axios.js` as well, the error seems to be happening inside there

**Reply by brogits** [reply]
awesome catch. you were right. i had an undeclared variable in axios. silly me.
thank you brother for the quick response.

all good for me now
